charts/spire/charts/spire-server/values.yaml

```diff
@@ -73,12 +73,29 @@ affinity: {}
 
 topologySpreadConstraints: []
 
-dataStorage:
+persistence:
+  # -- Enable persistence used by sqlite3 for spire-server, by the disk KeyStore, and/or by Tornjak.
   enabled: true
   size: 1Gi
   accessMode: ReadWriteOnce
   storageClass: null
 
+dataStore:
+  sql:
+    # -- Other supported databases are "postgres" and "mysql"
+    database_type: sqlite3
+    database: spire
+    host: ""
+    # -- likely 5432 for postgresql 3306 for mysql
+    port: 0
+    username: spire
+    password: ""
+    # -- Only used by "postgres" or "mysql"
+    options: []
```

Comment:
```suggestion
    options: []
```

Reply:
I can't find the comment now. probably lost in the squash, but I believe the comment there was requested by Marco.

Reply:
oh... I see what you did... this with the other patch puts a comment on the whole section instead...

But that isn't quite equivalent, because, while you can see the comment when looking a the raw values, it doesn't show up in the README. Not sure how to do the same thing in a way that would show up in the README

Reply:
Yea, not sure if thats possible with helm docs. Can alternatively add the only used by to each of the variable if you want it to show up in the readme.